payroll-processor-client/.prettierrc

```diff
@@ -1,11 +1,7 @@
 {
   "singleQuote": true,
   "trailingComma": "all",
-  "tabWidth": 2,
```

Comment:
This setting (and the other ones removed) is handled by the `.editorconfig` in the repo root.